Controller/MediaAdminController.php

```diff
@@ -64,7 +65,7 @@ public function browserAction()
         $formView = $datagrid->getForm()->createView();
 
         // set the theme for the current Admin Form
-        $this->get('twig')->getExtension('form')->renderer->setTheme($formView, $this->admin->getFilterTheme());
+        $this->setFormTheme($formView, $this->admin->getFilterTheme()[0]);
```

Comment:
Sonata 3.1 supports Symfony 2.3 which requires PHP >= 5.3.3. In any case, I don't understand this change, you have to pass the whole array here.